stubs/psutil/psutil/__init__.pyi

```diff
@@ -236,7 +236,7 @@ class Popen(Process):
 def pids() -> list[int]: ...
 def pid_exists(pid: int) -> bool: ...
 def process_iter(
-    attrs: list[str] | tuple[str, ...] | set[str] | frozenset[str] | None = None, ad_value=None
+    attrs: list[str] | tuple[str, ...] | set[str] | frozenset[str] | None = None, ad_value: object = None
```

Comment:
```suggestion
    attrs: list[str] | tuple[str, ...] | set[str] | frozenset[str] | None = None,
    ad_value: Any = None,  # arbitrary value used as result in case of error
```